msal/src/main/java/com/microsoft/identity/client/MultipleAccountPublicClientApplication.java

```diff
@@ -260,11 +270,14 @@ public void onTaskCompleted(final List<ICacheRecord> result) {
 
                                         for (final IAccount account : accounts) {
                                             if (accountMatcher.matches(trimmedIdentifier, account)) {
+                                                Logger.info(TAG + methodName,
+                                                        "Account found, Successful callback for identifier:" + identifier);
                                                 callback.onTaskCompleted(account);
                                                 return;
                                             }
                                         }
-
+                                        Logger.info(TAG + methodName,
+                                                "No accounts found callback for identifier:" + identifier);
```

Comment:
Here, can we actually dump the list of accounts that we didn't find anything for?

Reply:
added